constants/constants.go

```diff
@@ -95,3 +97,18 @@ const OTPEmailSubject = "One Time Password"
 
 // EmailMIME ...
 const EmailMIME = "MIME-version: 1.0;\nContent-Type: text/html; charset=\"UTF-8\";\n\n"
+
+// <------------- time tracker constants ------------>
+
+// GenericTimeTrackersList is list of generic time providers which can be used for any task provider.
+var GenericTimeTrackersList = []string{"gsheet"}
+
+// time trackers value and display name.
+const (
```

Comment:
No need for this to be here, we can create them in the tasktracker's provider file. Also, we should keep as less of our application specific code there.

Reply:
removed.